pkg/manifest/objects_test.go

```diff
@@ -0,0 +1,249 @@
+// Copyright 2019 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package manifest
```

Comment:
Thanks a lot for doing this Morven! 
One high level suggestion is to move these to a subtest pattern, see https://blog.golang.org/subtests.
This makes it easy to run just one failing test and in general helps with concurrency (though here it doesn't really matter). 
See here, it's just a couple of extra lines: https://github.com/istio/operator/blob/d98093ec44c1d59a6c42865b2dd50e988e42f758/pkg/patch/patch_test.go#L174

Reply:
Done.